src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        int index = new Random().nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/32c8c7a4-0043-4a0d-b912-d9c429c137a7)
The same about Random and ColorSupplier